lib/src/main/java/com/auth0/android/lock/utils/EnterpriseConnectionMatcher.java

```diff
@@ -44,20 +44,20 @@
     private static final String DOMAIN_ALIASES_KEY = "domain_aliases";
     private static final String AT_SYMBOL = "@";
 
-    private List<Strategy> strategies;
+    private List<Connection> connections;
 
-    public EnterpriseConnectionMatcher(List<Strategy> strategies) {
-        this.strategies = new ArrayList<>();
-        if (strategies == null) {
+    public EnterpriseConnectionMatcher(List<Connection> connections) {
+        this.connections = new ArrayList<>();
+        if (connections == null) {
             return;
         }
 
-        for (Strategy s : strategies) {
-            if (s.getType() == Strategies.Type.ENTERPRISE) {
-                this.strategies.add(s);
+        for (Connection s : connections) {
+            if (s.getType() == AuthType.ENTERPRISE) {
```

Comment:
@hzalaz It was a safe-filter to only use enterprise connections in this class. But the connections come from `configuration.getEnterpriseConnections()` so this can be removed. Also, the class can be renamed to something more generic like `ConnectionDomainMatcher`.